src/constants/i18n.ts

```diff
@@ -1,10 +1,12 @@
+import * as locales from '@/locales';
+
 type Language = {
-  key: string;
+  key: keyof typeof locales;
   dir?: 'ltr' | 'rtl';
   fontScale?: number;
 };
 
-export const DEFAULT_LANGUAGE_KEY: Language['key'] = 'en';
```

Comment:
It's not very important, but I think it's better to keep `Language['key']` here

Reply:
the issue is made by this, Language['key'] is string